src/rustdoc_schema.graphql

```diff
@@ -462,6 +471,9 @@ type Function implements Item & FunctionLike & Importable {
   unsafe: Boolean!
   async: Boolean!
 
+  # edges from FunctionLike
+  parameters: [FunctionParameter!]
```

Comment:
A couple minor nitpicks:
- Let's try to keep the schema elements in order parallel to the implemented interfaces followed by the "own" elements (properties / edges). This keeps it a bit more consistent and easier to read: `Item`'s elements always before `FunctionLike`'s ones, own at the end, etc.
- Let's keep edge names singular, because when used in a query, they only capture more than one element if we use an aggregation like `@fold`.

```suggestion
  # edges from FunctionLike
  parameter: [FunctionParameter!]
```